### PR TITLE
Commodities descriptions  english corrections

### PR DESCRIPTION
**Pull request by exoterws**
Correction and style updates to commodities descriptions. Overall meaning of the descriptions should not be changed enough to require new translations.

### CODE REVIEW

**Comment by exoterws** [comment]
Apologies, I don't know why changelog and mission.c are part of this commit.

File: CHANGELOG
Comment (exoterws) [inline CHANGELOG:1]: Should not be part of the PR

File: src/mission.c
Comment (exoterws) [inline src/mission.c:1]: should not be part of the PR

**Comment by bobbens** [comment]
Fixed the merge conflicts. Not sure what happened there. Thanks!
